### DISCUSSION TITLE
set icon

### DISCUSSION THREAD

**Discussion by xuyastc** [kyazdani42/nvim-web-devicons#178] [Q&A] [answered]
### Hi，I want.zshrc_xuya_config to use the same icon as.zshrc.How should override be written?
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/37319368/203471540-94bae29a-12b9-45c3-b970-860e62ef6e8f.png">

### The second question
<img width="252" alt="image" src="https://user-images.githubusercontent.com/37319368/203471597-647418a5-7fe9-45b7-b210-46b2290ad9bc.png">
How do I modify this?

**I need you help, thanks**

**Comment by xuyastc** [comment]
**I tried to change override but it didn't work**
so I need you help

**Comment by alex-courtis** [comment, marked as answer]
You can set that individual file to be a clone of .zshrc via something like:

```lua
  local icon, color, cterm_color = require'nvim-web-devicons'.get_icon_colors('.zshrc')

  require('nvim-web-devicons').set_icon({
    ['.zshrc_xuya_co'] = {
      icon = icon,
      color = color,
      cterm_color = cterm_color,
      name = "Zshrc",
    }
  })
```

**Comment by alex-courtis** [comment]
> How do I modify this?

What do you want to modify?
